The unauthenticated git protocol on port 9418 is no longer supported.

When installing the package, I get the following error:
```
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t git://github.com/agsh/onvif.git
npm ERR!
npm ERR! fatal: remote error:
npm ERR!   The unauthenticated git protocol on port 9418 is no longer supported.
npm ERR! Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
npm ERR!
npm ERR! exited with error code: 128
```

--- a/package.json
+++ b/package.json
@@ -20,7 +20,7 @@
     "commander": "^4.0.1",
     "nimble": "0.0.2",
     "node-datetime": "^2.1.2",
-    "onvif": "git://github.com/agsh/onvif.git#cc4e57f",
+    "onvif": "https://github.com/agsh/onvif.git#cc4e57f",
     "request-digest": "^1.0.13",
     "xml2js": "^0.4.22"
   },

